fontbe/src/orchestration.rs

```diff
@@ -705,7 +708,7 @@ mod tests {
     fn non_default_region() -> VariationRegion {
         let mut region = VariationRegion::default();
         region.insert(
-            "Weight".to_string(),
+            Tag::from_str("wght").unwrap(),
```

Comment:
I'd prefer writing this as,


```suggestion
            Tag::new(b"wght"),
```